frontend/app/src/views/Dashboard.vue

```diff
@@ -183,78 +174,73 @@ import { TaskType } from '@/types/task-type';
     ManualBalanceCardList,
     BlockchainBalanceCardList
   },
-  computed: {
-    ...mapState(useTasks, ['isTaskRunning']),
-    ...mapGetters('balances', [
-      'exchanges',
-      'manualBalanceByLocation',
-      'aggregatedBalances',
-      'liabilities',
-      'blockchainTotals',
-      'blockchainTotal'
-    ])
-  },
-  methods: {
-    ...mapActions('balances', [
-      'fetchExchangeBalances',
-      'fetchBlockchainBalances',
-      'fetchManualBalances',
-      'fetchLoopringBalances'
-    ])
-  }
-})
-export default class Dashboard extends Vue {
-  exchanges!: ExchangeInfo[];
-  isTaskRunning!: (type: TaskType) => Ref<boolean>;
-  blockchainTotals!: BlockchainTotal[];
-  aggregatedBalances!: AssetBalance[];
-  liabilities!: AssetBalance[];
-  manualBalanceByLocation!: LocationBalance[];
-  fetchBlockchainBalances!: (
-    payload: BlockchainBalancePayload
-  ) => Promise<void>;
-  fetchLoopringBalances!: (refresh: boolean) => Promise<void>;
-  fetchExchangeBalances!: (payload: ExchangeBalancePayload) => Promise<void>;
-  fetchManualBalances!: () => Promise<void>;
+  setup() {
+    const { isTaskRunning } = useTasks();
+    const {
+      blockchainTotals,
+      aggregatedBalances,
+      liabilities,
+      manualBalanceByLocation,
+      fetchBlockchainBalances,
+      fetchLoopringBalances,
+      fetchManualBalances
+    } = setupGeneralBalances();
 
-  get blockchainIsLoading(): boolean {
-    return get(this.isTaskRunning(TaskType.QUERY_BLOCKCHAIN_BALANCES));
-  }
+    const { exchanges, fetchExchangeBalances } = useExchanges();
 
-  get exchangeIsLoading(): boolean {
-    return get(this.isTaskRunning(TaskType.QUERY_EXCHANGE_BALANCES));
-  }
+    const isBlockchainLoading = computed<boolean>(() => {
+      return get(isTaskRunning(TaskType.QUERY_BLOCKCHAIN_BALANCES));
```

Comment:
Can we just use the isTaskRunning directly instead of wrapping to a computed ref? it should already return a computed ref right?